packages/docusaurus-plugin-content-docs/src/node/index.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// APIs available to Node.js
+export * from '../constants';
+export * from '../versions';
```

Comment:
for constants I'm fine with * but I'd rather have named exports for everything else, so that we don't expose publicly things by mistake